docs/advanced/working_with_git.rst

```diff
@@ -0,0 +1,207 @@
+================
+Working with git
+================
+
```

Comment:
It would be helpful to add an introduction sentence here. Perhaps something like:

OpenHatch has a long history of helping contributors build existing and learn new skills. This document section has information that OpenHatch contributors have found useful when working with git. We encourage you to share helpful git resources by adding your favorites to this file and creating a pull request.